torch/csrc/autograd/functions/accumulate_grad.cpp

```diff
@@ -30,10 +30,11 @@ auto AccumulateGrad::apply(const variable_list& grads) -> variable_list {
     new_grad = (*hook)({new_grad})[0];
   }
 
-  auto& grad = variable.grad();
-  if (!grad.defined()) {
+  auto& grad_tensor = variable.grad();
```

Comment:
Can we please keep it's name as `grad` and just make the type `at::Tensor`? We usually call things `_tensor` when they are not variables (because the type system doesn't help us), and this will be confusing